bytebase/bytebase.com
# feat: contact, refund and demo confirm pages

**This PR:**
Adding Contact Us page.

TODO:
- [ ] Add api and fetch to send form data to crm;
- [x] Add cal.com integration;
- [x] Develop Demo confirm section;
- [x] Develop refund section;
- [x] Update SEO;

**Steps to test**

- Open the [Contact us page](https://bytebase-git-contact-pages-bytebase-pixelpoint.vercel.app/contact-us/)
- Open the [Refund page](https://bytebase-git-contact-pages-bytebase-pixelpoint.vercel.app/refund/)
- Open the [Demo page](https://bytebase-git-contact-pages-bytebase-pixelpoint.vercel.app/demo-confirm/)
- Confirm that everything looks and works as expected
